mlbusinesscomponents/src/main/java/com/mercadolibre/android/mlbusinesscomponents/components/touchpoint/view/flex_cover_carousel/FlexCoverCarouselComponent.kt

```diff
@@ -28,24 +35,40 @@ class FlexCoverCarouselComponent @JvmOverloads constructor(
     private var wasAtLastElement: Boolean? = null
     private var normalPadding = 0
     private var biggerPadding = 0
+    private lateinit var tracker: MLBusinessTouchpointTracker
+    private lateinit var printProvider: TouchpointPrintProvider
+    private val recyclerRect by lazy { Rect() }
 
-    override fun bind(model: FlexCoverCarouselResponse, onClickCallback: OnClickCallback?) {
+    override fun bind(
+        model: FlexCoverCarouselResponse,
+        onClickCallback: OnClickCallback?,
+        tracker: MLBusinessTouchpointTracker?,
+        printProvider: TouchpointPrintProvider
+    ) {
         if (!isInitialized) {
             initialize()
         }
-        presenter.bind(model, onClickCallback)
+        presenter.bind(model, onClickCallback, tracker, printProvider)
     }
 
     override fun notifyPadding(additionalEdgeInsets: AdditionalEdgeInsets) {
         normalPadding = ScaleUtils.getPxFromDp(context, additionalEdgeInsets.left.toFloat()).toInt()
         biggerPadding = ScaleUtils.getPxFromDp(context, additionalEdgeInsets.right.toFloat()).toInt()
     }
 
+    override fun setTracker(tracker: MLBusinessTouchpointTracker?) {
+        tracker?.let { this.tracker = it }
+    }
+
+    override fun setPrintProvide(printProvider: TouchpointPrintProvider) {
```

Comment:
Sole, corrijo este typo
```suggestion
    override fun setPrintProvider(printProvider: TouchpointPrintProvider) {
```